pages/watches-support.json

```diff
@@ -561,7 +561,7 @@
       "status":"supported",
       "maintainers": [  "jrt", "MagneFire" ],
       "contributors": [ "flocke", "fosspill", "FlorentBrianFoxcorner", "jrt", "MagneFire"  ],
-      "stars":3,
+      "stars":4,
```

Comment:
Performance is really bad, it should not have an equal rating to for example `sturgeon`.

Reply:
fixed